Building-with-IntelliJ-IDEA.md

```diff
@@ -20,104 +21,138 @@ You can build RuneLite locally using [JDK 11](https://adoptium.net/temurin/relea
 
 So, to actually run RuneLite, we first need to invoke Maven.
 
-Locate *Maven* on right-side of the screen until you open something like this:
+Locate *Maven* on right-side of the screen and navigate to ``RuneLite`` (parent) > ``Lifecycle``,right click ``install``, and press ``Run Maven Build``:
+
+![run_maven_build.png](img%2Fbuilding-with-intellij%2Frun_maven_build.png)
+
+At this point, it should start the build, which might take a while. Once the process is done, you should see a message to let you know the final status of the build. Don't worry if there are build or test failures. We can fix those later.
+
+![successful_build.png](img%2Fbuilding-with-intellij%2Fsuccessful_build.png)
 
-![project-maven-view](https://i.imgur.com/xIRxBN1.png)
+It is highly recommended to create custom ``Run Configurations``. In the Maven window, right-click on ``RuneLite`` > ``Lifecycle`` > ``install``, and select ``Modify Run Configuration...``
 
-Standard maven goal is *Install* so let's run that. Locate *Runelite (root)* project in dropdown and navigate to `RuneLite (root) -> Lifecycle -> Install` and right-click the option and select `Run Maven Build`:
+![modify-run-configuration](img%2Fbuilding-with-intellij%2Fmodify-run-configuration.png)
 
-![run-maven-install](https://i.imgur.com/MxTMK6o.png)
+Here you can choose a name, such as ``runelite-parent [install] skip tests``, and add a Maven command. For faster builds, try ``install -DskipTests``. This will skip the tests (and test failures).
 
-If you are having issues with missing JDK see [Troubleshooting](#troubleshooting) section.
-You can now save this build step as *Run configuration*, just right click *Install* again and select *Create..* (*Modify Run Configuration...* in IntelliJ 2021.2), window like this should pop-up, so just hit *Ok* (you can append `-DskipTests` for faster builds):
+![install_skip_tests](img%2Fbuilding-with-intellij%2Finstall-skip-tests.png)
 
-![project-maven-install-save](https://i.imgur.com/vdJoJ7L.png)
+Press ``OK``. You can now easily run your run configurations in the Maven window by right-clicking ``RuneLite`` (parent) > ``Run Configurations`` > ``runelite-parent [install] skip tests`` and selecting ``Run``.  
+Alternatively, you can select your run configuration in the dropdown box at the top and hit the ``Run`` button.
 
-Now, we need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). So, still in same Maven window, just click the Refresh icon:
+![custom_configuration_top](img%2Fbuilding-with-intellij%2Fcustom-configuration-top.png)
 
-![project-maven-refresh](https://i.imgur.com/L2SmWtF.png)
+Other useful run configurations include:  
+``clean install -DskipTests -U``  which is your first troubleshooting step that you should run multiple times in case of a build failures  
```

Comment:
```suggestion
``clean install -DskipTests -U`` which is your first troubleshooting step that you should run multiple times in case of a build failures  
```